### PR TITLE
BOAC-2840: Includes appointments in note search

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2840

Appointments will start appearing in search results, but display will look weird until the UI work is done.

### CODE REVIEW

File: boac/api/search_controller.py [hunk 1]
```diff
@@ -174,10 +175,21 @@ def _notes_search(search_phrase, params):
...
+        advisor_csid=author_csid,
+        student_csid=student_csid,
+        topic=topic,
+        datetime_from=datetime_from,
+        datetime_to=datetime_to,
+        offset=offset,
+        limit=limit,
     )
 
+    combined_results = sorted(notes_results + appointment_results, key=lambda result: result['rank'], reverse=True)
```
Comment (johncrossman) [inline boac/api/search_controller.py]: @lyttam - Do the designs ask for merging notes and appts into a single list?

File: boac/api/search_controller.py [hunk 2]
```diff
@@ -174,10 +175,21 @@ def _notes_search(search_phrase, params):
         topic=topic,
         datetime_from=datetime_from,
         datetime_to=datetime_to,
-        offset=int(offset),
-        limit=int(limit),
+        offset=offset,
+        limit=limit,
+    )
+    appointment_results = Appointment.search(
```
Comment (johncrossman) [inline boac/api/search_controller.py]: @lyttam Feature flag needed here?

File: boac/api/search_controller.py (hunk 1 above)
Comment (lyttam) [reply boac/api/search_controller.py]: @johncrossman now that I look back at the design, it suggests they be in separate lists. I'll make that change.

File: boac/api/search_controller.py (hunk 2 above)
Comment (lyttam) [reply boac/api/search_controller.py]: @johncrossman yes, thank you. Will amend.

**Comment by johncrossman** [comment]
Thanks @lyttam  I'll merge when amends are in.

**Comment by lyttam** [comment]
@johncrossman Should be ready to merge.  One note:  I set the feature flag to True in test.py, any concerns with that?

**Comment by johncrossman** [comment]
Feature-flag=True in `test.py` is appropriate. Thanks @lyttam

**Pull request merged**